internal/validation/gateway/httproute.go

```diff
@@ -109,7 +109,7 @@ func validateHTTPRouteFeatures(httproute *gatewayv1alpha2.HTTPRoute) error {
 
 		// we don't support any backendRef types except Kubernetes Services
 		for _, ref := range rule.BackendRefs {
-			if ref.BackendRef.Group != nil && *ref.BackendRef.Group != "core" {
+			if ref.BackendRef.Group != nil && *ref.BackendRef.Group != "core" && *ref.BackendRef.Group != "" {
```

Comment:
I found this while testing in https://github.com/Kong/kubernetes-ingress-controller/pull/2243, we simply need to allow an empty string here because that will equate to "core".